packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -157,6 +165,29 @@ export class GuxAvatar {
     }
   }
 
+  private renderUcIntegrationsIcon(): JSX.Element | null {
+    switch (this.ucIntegration) {
+      case 'teams':
+        return renderTeamsSVG();
+      case 'zoom':
+        return renderZoomSVG();
+      case '8x8':
+        return render8x8SVG();
+      default:
+        return null;
+    }
+  }
+
+  private renderUcIntegrationBadge(): JSX.Element | null {
+    if (this.ucIntegration && this.size === 'large') {
+      return (
+        <div class="gux-avatar-integration-badge">
+          {this.renderUcIntegrationsIcon()}
+        </div>
+      ) as JSX.Element;
+    }
+  }
```

Comment:
Should we throw a warning if ucIntegration is true but size is not large?

Reply:
Added